matterbridge.toml.simple

```diff
@@ -1,7 +1,7 @@
 #WARNING: as this file contains credentials, be sure to set correct file permissions
 [irc]
     [irc.freenode]
```

Comment:
You'll want to change this line as well since you changed account to "irc.libera" elsewhere.

Reply:
You're right. I thought I already double checked, but it seems I was too confident…